docs/sources/configuration/_index.md

```diff
@@ -2467,6 +2467,10 @@ The `limits_config` block configures global and per-tenant limits in Loki.
 # Enable log-volume endpoints.
 [volume_enabled: <boolean>]
 
+# The maximum number of aggregated series in a log-volume response
+# CLI flag: -limits.volume-max-series
+[volume_max_series: <int> | default = 1000]
```

Comment:
could we use `max_query_series` as the limit instead? I like all the additional wiring in this PR to treat this as a different limit, but I'm curious if the underlying value should just come from `max_query_series` for now to avoid the additional configuration, and we can always easily change that later since it's all wired up as a separate limit?